data.json

```
@@ -1246,5 +1246,10 @@
     "githubUsername": "bhagatpratham",
     "imageUrl": "https://user-images.githubusercontent.com/66031743/128864527-177a97f6-967c-4b18-93d2-c02472274612.PNG",
     "issueNumber": 639
-  }
+  },
+  {
+    "name": "Gunjan Paneri",
+    "githubUsername": "iamgunjan",
+    "imageUrl": "https://user-images.githubusercontent.com/53567221/129094169-e012e3fb-3ba9-47e7-9173-0e09e7517034.png",
+    "issueNumber": 642
```

```suggestion
    "issueNumber": 642
  }
```
I think it should look something like this